pyrgg/graph_gen.py

```diff
@@ -90,38 +85,27 @@ def _write_data_to_json(buf, edge_dic, weight_dic):
 
 def _write_properties_to_json(
         buf,
-        min_weight,
```

Comment:
I have a question. Why did you remove `min_weight` and `max_weight` from the JSON format?

Reply:
Since they were only used for calculating for `weighted`, now it's passed to this function directly. Therefore we don't need them anymore. The reason was that `is_weighted` could be used when calling `gen_function` inside `gen_using`. Just for the case of consistency.